packages/core-contracts/contracts/proxy/BeaconProxy.sol

```diff
@@ -2,18 +2,22 @@
 pragma solidity ^0.8.0;
 
 import "./AbstractProxy.sol";
-import "./Beacon.sol";
 import "./BeaconStorage.sol";
 import "../common/CommonErrors.sol";
+import "../interfaces/IBeacon.sol";
 import "../utils/ContractUtil.sol";
 
 contract BeaconProxy is AbstractProxy, BeaconStorage, CommonErrors, ContractUtil {
+    error NoChange();
```

Comment:
Would this be better in CommonErrors?

Reply:
yeap..good point. I was wondering about the same. I left it there till we decide if we want to use a perhaps better term